src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -627,3 +629,65 @@ def find_similar_records(
             UserWarning,
         )
         return []
+
+    def compute_annotator_metrics(
```

Comment:
This looks good. perhaps we should also rename `unify_responses` to `compute_unified_responses`? I think aligning naming.